docs/providers/aws/events/streams.md

```diff
@@ -329,3 +329,22 @@ functions:
           arn: arn:aws:dynamodb:region:XXXXXX:table/foo/stream/1970-01-01T00:00:00.000
           tumblingWindowInSeconds: 30
 ```
+
+## Setting FilterCriteria
+
+This configuration allows customers to filter event before lambda invocation. It accepts up to 5 filter criterion. If one event matches at least 1 pattern, lambda will process it.
```

Comment:
Grammarly suggested changes:
```
This configuration allows customers to filter events before a function invocation. It accepts up to 5 filter criteria. If one event matches at least 1 pattern, the function will process it.
```

Additionally, maybe we should mention that it supports 5 by default and up to 10 with quota extension?